### DISCUSSION TITLE
Airflow Job Not running at max load

### DISCUSSION THREAD

**Discussion by immalhotra** [apache/airflow#32577] [Q&A]
Hi, 

Airflow version - 2.4.3

I wanted to run 40 parallel tasks in a dag, thus have set the parameter of max_active_runs = 40 for the dag. Although airflow is running 40 parallel task, but only 5 jobs get into the running state. Rest of the jobs are stuck at scheduled state.

I am running sequentialExecutor, and don't want to change into LocalExecutor in production, as this will take a lot of time. Please suggest a way to run all 40 jobs together.

Below is the screenshot for reference. Let me know if any other information is required.

<img width="1584" alt="Screenshot 2023-07-13 at 12 05 54 PM" src="https://github.com/apache/airflow/assets/12380943/4d8225bb-e190-452f-befc-40161d07d5d8">

**Comment by potiuk** [comment]
Copying my answer from Slack:

* DON'T use Sequential Excecutor in production. PERIOD
* It's not a matter what you want, you won't solve it without changing executor. Period. End of story
* It's very clearly stated in our documentation
* So don't argue with it, just change it to Local Executor. What you see is an inherent and unchangeable feature of Sequential executor.

You can't fight the reality
